src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -351,9 +352,11 @@ def load_arguments(self, _):
         # addons
         c.argument('enable_addons', options_list=['--enable-addons', '-a'])
         c.argument('workspace_resource_id')
-        c.argument('enable_msi_auth_for_monitoring', arg_type=get_three_state_flag(), is_preview=True)
-        c.argument('enable_syslog', arg_type=get_three_state_flag(), is_preview=True)
-        c.argument('data_collection_settings', is_preview=True)
+        c.argument('enable_msi_auth_for_monitoring', arg_type=get_three_state_flag())
+        c.argument('enable_syslog', arg_type=get_three_state_flag())
+        c.argument('data_collection_settings')
+        c.argument('azure_monitor_private_link_scope_resource_id', validator=validate_azuremonitor_privatelinkscope_resourceid)
```

Comment:
Since the name of this parameter is too long, could we provide a short alias for it